app/components/orkestrator-opplysning/OrkestratorOpplysningLand.tsx

```diff
@@ -0,0 +1,35 @@
+import { Select } from "@navikt/ds-react";
+import { FormScope, useField } from "@rvf/remix";
+
+import { useTypedRouteLoaderData } from "~/hooks/useTypedRouteLoaderData";
+import { IOrkestratorBarnOpplysning } from "~/models/orkestrator-opplysning.server";
+import { hentOrkestratorBarnOpplysningLabel } from "~/utils/orkestrator-opplysninger.utils";
+
+interface IProps {
+  opplysning: IOrkestratorBarnOpplysning;
+  formScope: FormScope<string>;
+}
+
+export function OrkestratorOpplysningLand({ opplysning, formScope }: IProps) {
+  const { orkestratorLandliste } = useTypedRouteLoaderData("routes/oppgave.$oppgaveId");
+  const field = useField(formScope);
+
+  // Todo: Finn ut hvordan vi skal håndtere readOnly på den her
+  // Det skal være mulig å endre barn oppholdsland uansett om det er fra register eller søknad
```

Comment:
Denne komponenten her har ikke foreløpig ikke en `readOnly` modus som alle andre. Oppholdsland for barn skal være mulig å endre av saksbehandler uansett om det kommer fra register.

Litt usikker på hvordan vi kan håndtere den her på en god måte.

Reply:
Går det an å sette value=land.alpha3kode på selve selecten, så får man ikke til å velge noe annet?

Reply:
Kan man sette value=land.alpha3kode rett på selecten når det er readonly?